docs/source/model_design/designing_vivarium_model/interactive_sim/index.rst

```diff
@@ -0,0 +1,116 @@
+..
+  Section title decorators for this document:
+  
+  ==============
+  Document Title
+  ==============
+  Section Level 1
+  ---------------
+  Section Level 2
+  +++++++++++++++
+  Section Level 3
+  ~~~~~~~~~~~~~~~
+  Section Level 4
+  ^^^^^^^^^^^^^^^
+  Section Level 5
+  '''''''''''''''
+
+  The depth of each section level is determined by the order in which each
+  decorator is encountered below. If you need an even deeper section level, just
+  choose a new decorator symbol from the list here:
+  https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections
+  And then add it to the list of decorators above.
+
+.. _vivarium_interactive_simulation:
+
+====================================
+Utilizing the Interactive Simulation
+====================================
+
+.. contents::
+   :local:
+   :depth: 2
+
+Intro to Interactive Simulations
+--------------------------------
+
+Interactive simulations are a way to step inside of a simulation intsead of 
+just receiving final outputs. This can be very helpful for a number of tasks 
+including V&V, debugging, and reviewing primary outputs. These are described 
+in detail :ref:`below <interactive_tasks>`. 
+
+The interactive sim allows a user to see all of the data available for any simulant. 
+For example, in the main simulation you might record the average BMI in the population 
+by age and sex. In the interactive simulation, you can see each simulant's individual 
+BMI as well as their propensity score for BMI. 
+
+In addition, you can move the simulation forward in time to watch and record how 
+simulants change and experience health events. 
+
+However, we tend to use much smaller populations for the interactive simulation compared to 
+the main simulation. This makes some tasks more difficult to complete interactively. 
```

Comment:
I wonder if it is worth including some explanation of _why_ this is? It's actually not super clear in my head.